Packs/CommonWidgets/Scripts/MyToDoTasksWidget/MyToDoTasksWidget.py

```diff
@@ -0,0 +1,67 @@
+from dateparser import parse
+
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def get_open_to_do_tasks_of_current_user() -> List[Dict]:
+    body = {
+        "dataType": "todos",
+        "query": "assignee:\"{me}\"",
+        "widgetType": "table"
+    }
+    todo_tasks_query_res = demisto.internalHttpRequest(
+        'POST',
+        '/v2/statistics/widgets/query',
+        json.dumps(body)
+    )
+
+    table = []
+
+    if todo_tasks_query_res.get('statusCode') == 200:
+        todo_tasks = json.loads(todo_tasks_query_res.get('body', '{}'))
+        todo_tasks_data = todo_tasks.get('data', [])
+        for task in todo_tasks_data:
+            if task.get('status', '') == 'open':  # table includes only open tasks
+                title = task.get('title', '')
+                description = task.get('description', '')
+                task_id = task.get('id', '')
+                incident_id = task.get('incidentId', '')
+                clickable_incident_id = f'[{incident_id}]({os.path.join("#/Custom/caseinfoid", incident_id)})'
```

Comment:
is `/Custom/caseinfoid` required? does it work properly in multi tenant? high availability?

Reply:
checked it on mt server and it worked fine.